storybook/README.md

```diff
@@ -0,0 +1,78 @@
+# Building Storybook with Webassembly and Qt 5.14
+
+
+
+## Configuring the environment
+### Install Emscripten v1.38.27
+	
+
+       # Get the emsdk repo
+	    git clone https://github.com/emscripten-core/emsdk.git
+	    
+	    #go to emsdk folder
+	    cd emsdk
+	    
+	    #install Emscripten v1.38.27
+	    ./emsdk install emscripten-1.38.27
+	    
+	    #activate emscripten-1.38.27
+	    ./emsdk activate emscripten-1.38.27
+	    
+	    #install Fastcomp backend
+	    ./emsdk install fastcomp-clang-tag-e1.38.27-64bit
+	    
+	    #activate Fastcomp backend
+	    ./emsdk activate fastcomp-clang-tag-e1.38.27-64bit
+	    
+	    #add emsdk tools to env variables
+	    ./emsdk construct_env
+	    echo 'source "/Users/alexjbanca/Repos/emsdk/emsdk_env.sh"' >> $HOME/.zprofile
```

Comment:
Don't reference your private dirs, use sth like `$HOME` or `~`

Reply:
Done. Added instructions to follow the instructions received by emsdk after running activate command and some insights of how it will affect the development environment.